image-index.md

```diff
@@ -1,12 +1,12 @@
-# OCI Image Manifest List Specification
+# OCI Image Index Specification
 
-The manifest list is a higher-level manifest which points to specific [image manifests](manifest.md) for one or more platforms.
-While the use of a manifest list is OPTIONAL for image providers, image consumers SHOULD be prepared to process them.
+The Image Index is a higher-level manifest which points to specific [image manifests](manifest.md), ideal for one or more platforms.
```

Comment:
Title-case "Image Index" here (you seem to prefer the sentence-case "image index).